src/commands/general/docsearch.ts

```diff
@@ -0,0 +1,54 @@
+import Filter from 'bad-words';
+import { Colors } from 'discord.js';
+import { CommandDefinition } from '../../lib/command';
+import { CommandCategory } from '../../constants';
+import { makeEmbed } from '../../lib/embed';
+
+export const docsearch: CommandDefinition = {
+    name: ['docsearch', 'documentation', 'doc'],
+    description: 'Provides a link to the documentation or documentation search for a quick link if there is no dedicated command.',
+    category: CommandCategory.GENERAL,
+    executor: (msg) => {
+        const searchWords = msg.content.replace(/^\.(doc|documentation|docsearch)\s+/, ' ').split(' ');
+
+        if (searchWords.length <= 1) {
+            const noQueryEmbed = makeEmbed({
+                title: 'FlyByWire Documentation',
+                description: 'Find the full FlyByWire Documentation [here](https://docs.flybywiresim.com/).',
+            });
+            return msg.channel.send({ embeds: [noQueryEmbed] });
+        }
+
+        console.log(searchWords);
+        // Safety to prevent users from sending their own links in bot output.
+        for (const searchWord of searchWords) {
+            try {
+                const _ = new URL(searchWord);
+                const URLEmbed = makeEmbed({
+                    title: 'FlyByWire Documentation | Error',
+                    description: 'Providing URLs to the Documentation search command is not allowed.',
+                    color: Colors.Red,
+                });
+                return msg.channel.send({ embeds: [URLEmbed] });
+            } catch (_) { /**/ }
+
+            const filter = new Filter();
+            if (filter.isProfane(searchWord)) {
+                return msg.reply('Please do not use profane language with this command.');
+            }
+        }
+
+        // Removing first element as it is an empty string
```

Comment:
Need this comment?

Reply:
No, just wanted to clarify if anyone was wondering. Happy to remove it

Reply:
Removed it, so should be resolved.